.prettierignore

```diff
@@ -2,6 +2,7 @@ package.json
 package-lock.json
 /__snapshots__
 /.next
+/.cache
```

Comment:
👍 I just noticed prettier was looking at this directory, glad it's getting fixed!